Change the name from tidy5 back to tidy.
https://github.com/htacg/tidy-html5/issues/46

Don't run bitness check which isn't needed except on Windows.
https://github.com/htacg/tidy-html5/issues/192

Change the mandir from man to share/man.
https://github.com/htacg/tidy-html5/issues/193
--- CMakeLists.txt.orig	2015-04-11 08:25:07.000000000 -0500
+++ CMakeLists.txt	2015-04-13 07:29:39.000000000 -0500
@@ -2,7 +2,7 @@
 # Prepare for changing the name to 'tidy'
 cmake_minimum_required (VERSION 2.8.8)
 
-set(LIB_NAME tidy5)
+set(LIB_NAME tidy)
 
 project (${LIB_NAME})
 
@@ -24,12 +24,14 @@
 option( BUILD_TAB2SPACE  "Set ON to build utility app, tab2space" OFF )
 option( BUILD_SAMPLE_CODE "Set ON to build the sample code"       OFF )
 
+if(WIN32 AND MSVC)
 if (CMAKE_SIZEOF_VOID_P EQUAL 8)
     message(STATUS "*** Have SIZEOF void * = 8, so 64-bit")
     set( IS_64_BIT 1 )
 else ()
     message(STATUS "*** SIZEOF void * != 8, so not 64-bit")
 endif ()
+endif()
 
 if(CMAKE_COMPILER_IS_GNUCXX)
     set( WARNING_FLAGS -Wall )
@@ -223,7 +225,7 @@
             VERBATIM
         )
 
-        install(FILES ${CMAKE_BINARY_DIR}/${TIDY_MANFILE} DESTINATION man/man1)
+        install(FILES ${CMAKE_BINARY_DIR}/${TIDY_MANFILE} DESTINATION share/man/man1)
     endif ()
 endif ()
 
